_posts/izayak/2016-04-05-MOAR-ry.md

```diff
@@ -0,0 +1,20 @@
+---
+author: izayak
+title: "Ruotong's MOAR Dictionary Exercises"
+layout: post
+---
+
+** Exercise 1 **  
+<iframe src="https://trinket.io/embed/python3/e969cf4951" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>  
+The first exercise is like the examples in the textbook (if I understood it correctly). At first I just made a dictionary of all the words. After I found there are both "you" and "You", and "\\{em" in it, I added two lines to remove the capital letters and punctuations.
+
+  
+** Exercise 2-5 **  
+<iframe src="https://trinket.io/embed/python3/e54d5c422e" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>  
```

Comment:
Nice work.  By splitting things into lines you made your regexing easier.  Make sure to understand how you could regex the whole file as well. (see my solution)